R/run_example.R

```diff
@@ -1,457 +1,458 @@
-              checkboxIds = c("op1", "op2", "op3"),
-              label = "Which types of waste do you transport?",
-              hint_label = "Select all that apply."),
-            heading_text("select_Input", size = "s"),
-            select_Input(
-              inputId = "sorter",
-              label = "Sort by",
-              select_text = c("Recently published",
-                              "Recently updated",
-                              "Most views",
-                              "Most comments"),
-              select_value = c("published", "updated", "view", "comments")),
-            heading_text("file_Input", size = "s"),
-            file_Input(inputId = "file1", label = "Upload a file"),
-            heading_text("button_Input", size = "s"),
-            button_Input("btn1", "Go to next page")
-          )
-        ),
+                heading_text("radio_button_Input (inline)", size = "s"),
+                radio_button_Input(
+                  inputId = "name_changed", label = "Have you changed your name?",
+                  choices = c("Yes", "No"), inline = TRUE,
+                  hint_label = "This includes changing your last name or spelling
+                            your name differently."
+                ),
+                heading_text("radio_button_Input (stacked)", size = "s"),
+                radio_button_Input(
+                  inputId = "name_changed_stacked", label = "Have you changed your name?",
+                  choices = c("Yes", "No"), inline = FALSE,
+                  hint_label = "This includes changing your last name or spelling
+                            your name differently."
+                ),
+                heading_text("checkbox_Input", size = "s"),
+                checkbox_Input(
+                  inputId = "checkID",
+                  cb_labels = c(
+                    "Waste from animal carcasses",
+                    "Waste from mines or quarries",
+                    "Farm or agricultural waste"
+                  ),
+                  checkboxIds = c("op1", "op2", "op3"),
+                  label = "Which types of waste do you transport?",
+                  hint_label = "Select all that apply."
+                ),
+                heading_text("select_Input", size = "s"),
+                select_Input(
+                  inputId = "sorter",
+                  label = "Sort by",
+                  select_text = c(
+                    "Recently published",
+                    "Recently updated",
+                    "Most views",
+                    "Most comments"
+                  ),
+                  select_value = c("published", "updated", "view", "comments")
+                ),
+                heading_text("file_Input", size = "s"),
+                file_Input(inputId = "file1", label = "Upload a file"),
+                heading_text("button_Input", size = "s"),
+                button_Input("btn1", "Go to next page")
```

Comment:
takes you to halfway down feedback page, expect to be taken to text types instead

Reply:
We had a few conflicting observes so made it go to text types now (though haven't sorted it going to the top of the page, it just stays how far you'd scrolled down on the page)